File: Gestion_Angular_Front_end/src/app/classes/lignefacture-dto.ts
import { FactureDto } from './facture-dto';
import { ProduitDto } from "./produit-dto";

export class LignefactureDto {
  push(ligneItem: LignefactureDto) {
    throw new Error('Method not implemented.');
  }
  id: number= 0;
  quantite: number= 0;
  //date: string= "";
  total: number=0;
 produitDto: ProduitDto= new ProduitDto();
 factureDtoDto: FactureDto= new FactureDto();

}

File: Gestion_Spring_Back_end/src/main/java/com/odji/spring_back_end/dto/LigneFactureDto.java
package com.odji.spring_back_end.dto;

import com.odji.spring_back_end.model.Facture;
import com.odji.spring_back_end.model.LigneFacture;
import com.odji.spring_back_end.model.Produit;
import jakarta.persistence.Column;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import lombok.*;

import java.math.BigDecimal;
import java.util.Date;
@Data
@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class LigneFactureDto {

    private Integer id;

    private ProduitDto produitDto;

    private FactureDto factureDto;

    private BigDecimal quantite;

    private String date;

}

